### PR TITLE
Implement aepp support using SDK

### LINKED ISSUE #277: Implement interface for SDK
Child of #276
For allowing an application to work inside the base app

### LINKED ISSUE #279: Create a screen for contract method call
Child of #276

### PR DESCRIPTION
**Pull request by davidyuk**
Closes #277, closes #279, related to #276

### CODE REVIEW

**Comment by davidyuk** [comment]
e2e tests are broken because of https://github.com/aeternity/aepp-sdk-js/issues/197

**Comment by thepiwo** [comment]
what I tried so far works well, there are a few issues where we still need to see if they are within the drone aepp oder integration

**Comment by kenodressel** [comment]
Hey everyone,
we (drone team) had a look at the issues and from our side with some small tweaks it looks good. 
:+1: for the merge.

File: src/lib/appsRegistry.js [hunk 1]
```diff
@@ -22,4 +22,8 @@ export const appsRegistry = {
     iconFullSize: true,
     path: 'addresses',
   },
+  6: {
```
Comment (thepiwo) [inline src/lib/appsRegistry.js]: Maybe we should exclude this from the PR for now

**Comment by davidyuk** [comment]
Costs of contract deploy and call of contract method are missed in this PR.

**Comment by davidyuk** [comment]
replaced by #344

**Pull request closed without merging**